### ISSUE TITLE
app.html clears after compilation

### ISSUE THREAD

**Issue by ErriourMe** [gaven/nuxt-svg-sprite-module#15]
I began to notice the problem when there are too many files, app.html is simply cleared during compilation without errors. If there is no previously generated sprite, then everything works as it should. Why is that?

**Comment by gaven** [comment]
Hey @ErriourRU - I'm not sure what you're referring to here. Are you saying that the `app.html` is deleted during compile time? If so, is this happening in development, or when you're statically generating a site?

**Comment by ErriourMe** [comment]
Yes, file is still exist but empty. It is happening after build production version (`npm run build`)
